src/Microsoft.AspNetCore.SystemWebAdapters.SessionState/BuiltIn/BuiltInSessionExtensions.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.AspNetCore.SystemWebAdapters.SessionState.BuiltIn;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters;
+
+public static class BuiltInSessionExtensions
+{
+    public static ISystemWebAdapterBuilder AddBuiltInSession(this ISystemWebAdapterBuilder builder, Action<Builder.SessionOptions>? options = null)
```

Comment:
BuiltIn is a non-descript API name. Rename all the BuiltIn types.
```suggestion
    public static ISystemWebAdapterBuilder WrapAspNetCoreSession(this ISystemWebAdapterBuilder builder, Action<Builder.SessionOptions>? options = null)
```